source/includes/assets-preview.md

```diff
@@ -0,0 +1,43 @@
+# Assets (Preview)
+
+```javascript
+const assets = require('assets-preview');
+```
+
+The Assets API can be used to fetch `AssetData` based on Ids, Names, and Tags.
+
+## Methods
+
+```javascript
+// Get all Assets available to this experience.
```

Comment:
The convention we've been following with slate is _generally_ to pull comments like this out into:

```
> Get all assets available...

` ` `
var all = assets.all();
` ` `
```

It looks nicer...